modules/cli-manager-adding-plugin-yaml.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-adding-plugin-yamls_{context}"]
+= Adding plugins to the {cli-manager} with YAML files
+
+You can add a CLI plugin to the {cli-manager} using the Form View or YAML View.
+
+[NOTE]
+====
+If you use the form view, add the information for the CLI plugin into the required fields, which are *Name*, *Platforms*, *Files*, *Image*, *Platform*, *Short description*, and *Version*. Then click *Create* to add the CLI plugin.
+====
+
+.Prerequisites
+
+* You are logged in to {product-title} as a user with the `cluster-admin` role.
+* The {cli-manager} is installed.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Select the *{cli-manager}* from the list.
+
+. Select the *CLI Plugin* tab.
+
+. Click *Create Plugin*.
+
+. Enter the YAML file with the information for the plugin you are installing in the text box. See the example YAML file below.
++
+.Example YAML file to add a plugin
+
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: Plugin
+metadata:
+  name: <plugin_name>
+spec:
+  shortDescription: short description of the <plugin_name>
+  description: long description of the <plugin_name>
+  version: <version> <1>
+  caveats: <2>
+  homepage: <plugin_homepage>
+  platforms:
+  - platform: <platform> <3>
+    image: <plugin_image>
+    imagePullSecret: <4>
+    files:
+    - from: <plugin_file_path>
+----
+<1> Version must be in v0.0.0 format.
+<2> This field is optional.
+<3> Add the architecture for your system. Ex. Linux/amd64, darwin/arm64, windows/amd64, or another architecture.
+<4> This field is for adding a pull secret to access your plugin image and is optional.
```

Comment:
@cbippley do not think this is optional, this is a mandatory parameter.

Reply:
Are y'all talking about #2 or #4?

Reply:
@cbippley the 4th callout. Perhaps could say something like:

```suggestion
<4> Optional: If the registry is not public, add a pull secret to access your plugin image.
```

Reply:
@cbippley agree with what andrea is saying here.